test/Gov.ts

```diff
@@ -118,7 +118,10 @@ describe("Gov", function () {
       // const supply =  await nft.totalSupply();
       // const result = supply.toNumber() * quorum / 100
       // console.log("result:", result)
-      expect(await gov.quorum(blockNumber - 1)).to.equal(2);
+      const today =  Math.floor(new Date().getTime() / 1000);
```

Comment:
Scotch tape found!
line 123 console log could be removed in pr.
Another idea would be to use verbosity levels in tests, to not produce noise in the cli output